wezterm-gui/src/termwindow/mouseevent.rs

```diff
@@ -425,6 +425,9 @@ impl super::TermWindow {
                 }
                 _ => {}
             },
+            WMEK::VertWheel(n) => {
+                self.activate_tab_relative(-n as isize, true).ok();
```

Comment:
The wheel movement may be scaled to larger than 1 based on inertia or screen scaling, which can make it difficult to "tick" through the tabs one by one.

For example, on my X11 system, wheel up events tend to tick by three each time whereas wheel down events tick down by one initially.  Weird but true!

I think this should probably clamp to a single movement per event, something like this:

```suggestion
                self.activate_tab_relative(if n < 1 { -1 } else { 1 }, true).ok();
```

Reply:
SGTM